nodejs/trivialOdbc/main.js

```diff
@@ -0,0 +1,15 @@
+const odbc = require('odbc');
+
+async function test() {
+  try {
+    const connection = await odbc.connect('DSN=LUGDEMO');

```

Comment:
@edmundreinhardt 

For the DSN I think we should use `*LOCAL` here because this DSN is preinstalled by IBM and uses the current user to establish the connection.

Reply:
I did it this way so that I could make it work locally on my PC as well as on the IBM i and show that I can develop locally and then deploy.
Would *LOCAL work for when I am on my PC.  I kind of doubt it.

Reply:
I see why you setup that way now. Well technically `*LOCAL` could work on your local machine if you setup a DSN in your `~/.odbc.ini` on your local machine:

```ini
[*LOCAL]
Description            = The IBM i System
Driver                 = IBM i Access ODBC Driver
System                 = PUT.YOUR.SYSTEM.HERE
UserID                 = USERNAME
Password               = PASSWORD
```

 or by setting up a DSN with `ODBC Data Source Administrator` tool on windows.

Reply:
I'm with @edmundreinhardt on this one. 
If the goal is to built and deploy cross-platform (PC and IBM i), one can either:
1. Customize a data source on IBM i (in this example, `LUGDEMO`) to match a DSN on the PC
2. Configure a `*LOCAL` data source on the PC. 

While having `*LOCAL` on the PC should work fine, it's a bit misleading (since it's talking to a remote system). I prefer the first option, with instructions saying "configure a datasource named XXXXX". One can easily copy the `*LOCAL` data source into a newly-named thing.

Reply:
OK Sounds good, I'm marking this as resolved now.